readme.md

```diff
@@ -80,244 +81,476 @@ As you're reviewing the app, try to fill in the blanks in the below Rails-to-Exp
 
 Write down **up to three questions** on topics you would like further clarification on. We will spend 15 minutes going over this (and the MVC chart) afterwards.
 
+### Note
+
+You may notice that we do things a bit differently in this example than we have in previous classes. For example, we use promises instead of callbacks when making Mongoose queries.
+
+This is to demonstrate that there is no single way of creating a Mongoose-Express app. If any of these differences seem confusing, don't hesitate to ask an instructor about them!
+
 ## Before We Continue!
 
 You are welcome to code along during the "I Do's" and "We Do's" in this lesson plan. We do ask, however, that **if you fall behind, do not attempt to catch up during those sections**. Instead, tilt down your screen and watch / take notes.
 
 You are more than welcome to catch up when we get to the "You Do's," during which the instructors are available to help.
 
------
-# Walkthrough
+## Mongoose
 
-# npm install --save mongoose
-
-> [e3ac57c](https://www.github.com/ga-wdi-exercises/whenpresident/commit/e3ac57c)
+### Why are we using Mongoose?
 
 Like ActiveRecord for Rails, Mongoose is an ORM we can use to represent data from a Mongo database as models in a Javascript back-end.
 
-### [npm install --save mongoose: `package.json`](https://www.github.com/ga-wdi-exercises/whenpresident/blob/e3ac57c/package.json)
-![npm install --save mongoose, package.json](_DIFFSHOTS/npm-install-save-mongoose.package-json.png)
+## Connect to Mongoose (10 minutes / 0:45)
 
-# Connected to Mongoose
+In order for us to use Mongoose to communicate with our database, we need to link it up to our Express application. We'll do this by...
+* Establishing a connection with a Mongo database.
+* Define a Mongoose schema and model.
+* Generate some seed data.
 
-> [1b8ff4f](https://www.github.com/ga-wdi-exercises/whenpresident/commit/1b8ff4f)
+### Steps
 
-#### Questions
+1. Install Mongoose via the command line: `npm install --save mongoose`.
```

Comment:
remember to `$ mongod` what does the error look like if you run connection.js, and the server isn't running?